ChemoTracker/App.js

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Platform } from 'react-native';
+import { NavigationComponent } from 'react-native-material-bottom-navigation';
+import { TabNavigator } from 'react-navigation';
 import Home from './components/Home.js';
+import Track from './components/Track.js';
+import Profile from './components/Profile.js';
+import Search from './components/Search.js';
+import Calendar from './components/Calendar.js';
 
-class App extends React.Component {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Home />
-      </View>
-    );
+const App = TabNavigator({
+  Home: { screen: Home, path: '' },
+  Track: { screen: Track, path: 'track' },
+  Search: { screen: Search, path: 'search' },
+  Calendar: { screen: Calendar, path: 'calendar' },
+  Profile: { screen: Profile, path: 'profile' }
+}, {
+  tabBarComponent: NavigationComponent,
+  tabBarPosition: 'bottom',
+  tabBarOptions: {
+    bottomNavigationOptions: {
+      labelColor: '#4f4f4f',
```

Comment:
For colours, we should create global, extern variables, so refactoring would be easier. https://stackoverflow.com/questions/37839755/is-there-a-way-of-storing-color-hex-codes-in-variables-similar-to-sass-in-react Here's some ideas of ways it could be done. This way way we all can just reference the variable name instead of manually typing in the hex everywhere